app/templates/admin/add_tags.html

```diff
@@ -15,85 +16,49 @@ <h2 class="ui header">
             </h2>
 
             <div class="ui menu">
+                <div id="select-role" class="ui dropdown item">

```

Comment:
why is there an add-tags file? as far as I see this is never called. we should just have view_tags and add_tag!

Reply:
sorry idk why thats still there i thought i deleted it thanks for pointing that out!